.github/actions/git-tag/action.yml

```diff
@@ -0,0 +1,33 @@
+name: "Publish new tag in a repo"
+description: "Publish new tag in a repo"
+
+inputs:
+  github_token:
+    description: 'Github token'
+    required: true
+    type: string
+  dry-run:
+    description: dry run flag
+    required: true
+    type: boolean
+
+runs:
+  using: "composite"
+  steps:
+    - name: publish tag
+      shell: bash
+      run: |
+        if [[  $BRANCH_NAME == "master" ]]; then
+          VERSION=$(grep -m1 version package.json | awk '{ print $2 }' | sed 's/[", ]//g')
+        fi;
+
+        echo "git tag -a ${VERSION} -m ${VERSION}"
+        if [[ ${{ inputs.dry-run }} == false ]]; then
+          git config --local user.name "alfresco-build"
+          git config --local user.email "build@alfresco.com"
```

Comment:
use https://github.com/Alfresco/alfresco-build-tools#configure-git-author so you don't hardcode anything (that email is not the current one anymore)

Reply:
Sure changed